### PR TITLE
Update minikube to v1.8.2

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@Madhu-1 @nixpanic ptal.

File: .travis.yml [hunk 1]
```diff
@@ -85,10 +85,10 @@ jobs:
         # - k8s-csi sidecar images for Arm64 are not available
         # - Travis Arm64 CI job runs inside unprivileged LXD which blocks
         #   launching minikube test environment
-        - travis_terminate 0    # deploy only on x86
+        - travis_terminate 0 # deploy only on x86
 
 deploy:
   - provider: script
-    on:  # yamllint disable-line rule:truthy
+    on: # yamllint disable-line rule:truthy
```
Comment (Madhu-1) [inline .travis.yml]: CI may fail here.

Comment (Madhu-1) [reply .travis.yml]: https://travis-ci.org/github/ceph/ceph-csi/jobs/663800182#L490-L494

Comment (humblec) [reply .travis.yml]: yeah, reverted.

**Comment by humblec** [comment]
> The commit message should mention that this if for Travis CI only.

Done. @nixpanic

**Comment by Madhu-1** [comment]
@humblec CI failure

File: .travis.yml [hunk 2]
```diff
@@ -85,7 +85,7 @@ jobs:
         # - k8s-csi sidecar images for Arm64 are not available
         # - Travis Arm64 CI job runs inside unprivileged LXD which blocks
         #   launching minikube test environment
-        - travis_terminate 0    # deploy only on x86
+        - travis_terminate 0  # deploy only on x86
```
Comment (Madhu-1) [inline .travis.yml]: please revert back this one also

Comment (humblec) [reply .travis.yml]: @Madhu-1 done

**Comment by humblec** [comment]
@nixpanic @Madhu-1 ptal

**Comment by nixpanic** [comment]
You'll need to add something like this:

```diff
--- a/scripts/minikube.sh
+++ b/scripts/minikube.sh
@@ -94,7 +94,7 @@ up)
     enable_psp
 
     echo "starting minikube with kubeadm bootstrapper"
+    minikube addons disable storage-provisioner || true
     # shellcheck disable=SC2086
     minikube start --memory="${MEMORY}" -b kubeadm --kubernetes-version="${KUBE_VERSION}" --vm-driver="${VM_DRIVER}" --container-runtime="${CONTAINER_RUNTIME}" --feature-gates="${K8S_FEATURE_GATES}" ${EXTRA_CONFIG}
 
```

**Comment by humblec** [comment]
@nixpanic done.. thanks. lets wait for CI.

**Comment by humblec** [comment]
Still failing.

File: scripts/minikube.sh [hunk 3]
```diff
@@ -93,6 +93,7 @@ up)
     enable_psp
 
     echo "starting minikube with kubeadm bootstrapper"
+    minikube addons disable storage-provisioner || true
```
Comment (Madhu-1) [inline scripts/minikube.sh]: looks like its failing https://travis-ci.org/github/ceph/ceph-csi/jobs/665699624#L379

Comment (Madhu-1) [reply scripts/minikube.sh]: can you disable default-storageclass` and `storage-provisioner` and check?

Comment (nixpanic) [reply scripts/minikube.sh]: It seems that I had to disable the EXTRA_OPTIONS (PodSecurityPolicy) as well to make it work.

Comment (Madhu-1) [reply scripts/minikube.sh]: In that case we need to open an issue in minikube to track it. Disabling psp is not a best option as we are good now with current version of minikube

**Comment by Madhu-1** [comment]
filed a minikube issue https://github.com/kubernetes/minikube/issues/7314

**Comment by Madhu-1** [comment]
moving out of release

**Comment by nixpanic** [comment]
@Madhu-1 is planning to work on #1027 so he should probably be assigned to this PR as well.

**Comment by humblec** [comment]
> @Madhu-1 is planning to work on #1027 so he should probably be assigned to this PR as well.

I could update the minikube .. no worries.

**Comment by Madhu-1** [comment]
minikube updated to latest version closing this now

**Pull request closed without merging**